### PR TITLE
Create TERMSOFUSE.txt

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Creates terms of use

# Needed By (Date)
- ?

# Urgency
- ?

# Steps to Test

1.  Read terms
2. Make edits

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: TERMSOFUSE.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Use of the Stanford-branded themes is limited to Stanford University academic and administrative activities. Use is granted solely to:
+
+* University academic departments, centers and research groups;
+* University administrative units;
+
+The Stanford brand bar and Stanford universal footer should not be edited or altered in any way, following the Identity toolkit guidelines at https://identity.stanford.edu/web-toolkit/
+
+Permission to use the work provided in this package is on a per-site basis. You will need to make a separate request for each website on which you would like to use these styles.
+
+For more information about policies related to the Stanford University identity guidelines, please see http://identity.stanford.edu/policies/
```
Comment (JBCSU) [inline TERMSOFUSE.txt]: The correct URL is https://identity.stanford.edu/policies.html

File: TERMSOFUSE.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Terms of Use
```
Comment (JBCSU) [inline TERMSOFUSE.txt]: This is a `.txt` file. Should it be `.md`?

File: TERMSOFUSE.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Stanford University owns all Stanford-branded themes and governs their use to benefit academic, research, communication and other University activities and interests. The University has the right to protect the use of these themes, the University name and University identity marks such as its seal, logos and symbols in all situations.
+
+Use of the Stanford-branded themes is limited to Stanford University academic and administrative activities. Use is granted solely to:
+
+* University academic departments, centers and research groups;
+* University administrative units;
+
+The Stanford brand bar and Stanford universal footer should not be edited or altered in any way, following the Identity toolkit guidelines at https://identity.stanford.edu/web-toolkit/
```
Comment (JBCSU) [inline TERMSOFUSE.txt]: I think the correct url is https://identity.stanford.edu/web-mobile.html#build-something-new. Not sure about including the anchor, tho.

File: TERMSOFUSE.txt (hunk 2 above)
Comment (sherakama) [reply TERMSOFUSE.txt]: The original file was in txt and I did not feel like adding in the additional formatting that md should have. I don't feel strongly about either format so I am leaning towards the path of least effort.

**Comment by sherakama** [comment]
@JBCSU Links are updated and note about file format left.

**Comment by sherakama** [comment]
@stocker this is ready for your review.

File: TERMSOFUSE.txt (hunk 2 above)
Comment (JBCSU) [reply TERMSOFUSE.txt]: Roger that.

**Pull request merged**